### PR TITLE
Luw 40 Refactor skillssection

### PR DESCRIPTION
**Pull request by fannymartensson**
- Refactored data structure.
- Refactored animation.
- Added refactored data json file to look at when added to other instances.
- Added new data to other instances in serverless cloud.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -3,96 +3,316 @@
...
+      "profileImage": "https://brilliant-code-hfv1j.cloud.serverless.com/public/images/rasa.jpg",
+      "id": "3",
+      "name": "Rasa Golali",
+      "gif": "https://brilliant-code-hfv1j.cloud.serverless.com/public/gifs/gif-rasa.gif"
+    },
+    "created": "2022-09-13T08:34:36.000Z",
+    "modified": "2022-09-19T13:02:37.000Z"
+  },
+  {
+    "key": "icon",
```
Comment (mariahnoren) [inline data.json:99]: Could teamMember- and icon-data be split into two separate .json files?

[the remark this answers is not in the document]
Comment (fannymartensson) [reply data.json:99]: This json is  only the json data to import/export data inside serverless cloud data. Therefore it displays in the same file.

Comment (fannymartensson) [reply data.json:99]: This json is only to import/export data inside serverless cloud data. Therefore it displays in the same file.

File: src/pages/HomePage/types.ts [hunk 2]
```diff
@@ -1,8 +1,7 @@
-import { Icon } from '../../components/Sections/Skills/types';
 import { SlsDbItem } from '../../types';
 
 export type Skill = SlsDbItem & {
-  techIcons: Icon[];
-  helperIcons: Icon[];
-  designIcons: Icon[];
+  alt: string;
+  id: number;
+  url: string;
```
[the remark this answers is not in the document]
Comment (fannymartensson) [reply src/pages/HomePage/types.ts]: Yes of course!

**Review by mariahnoren — APPROVED** [review]
Nice!

**Pull request merged**